scripts/webvid/plot.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 MosaicML Streaming authors
+# SPDX-License-Identifier: Apache-2.0
+
+"""Plot download performance comparison."""
+
+from argparse import ArgumentParser, Namespace
+
+import numpy as np
+from matplotlib import pyplot as plt
+
+
+def parse_args() -> Namespace:
+    """Parse command-line arguments.
+
+    Returns:
+        Namespace: Command-line arguments.
+    """
+    args = ArgumentParser()
+    args.add_argument('--out', type=str, required=True)
```

Comment:
Can you please add a `help` parameter on what each of the parameter means? Thanks!